actionpack/lib/abstract_controller/helpers.rb

```diff
@@ -84,10 +84,11 @@ def helper_method(*methods)
         file, line = location.path, location.lineno
 
         methods.each do |method|
+          # def current_user(*args, &block)
+          #   controller.send(:'current_user', *args, &block)
+          # end
           _helpers_for_modification.class_eval <<~ruby_eval, file, line
-            def #{method}(*args, &block)                    # def current_user(*args, &block)
-              controller.send(:'#{method}', *args, &block)  #   controller.send(:'current_user', *args, &block)
```

Comment:
This sounds a lot like `Module#delegate`, can't we call that instead?

Reply:
Actually nevermind, `delegate` wouldn't allow to pass the source location, and also does some extra things we don't need.